src/System.Text.Json/roadmap/images/PlantUML-license.svg

```diff
@@ -0,0 +1,15 @@
ll="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="537" x="5" y="665.9512">There is an exception : if the textual description in PlantUML language is also covered</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="474" x="5" y="679.9199">by a license (like the GPL), then the generated images are logically covered</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="158" x="5" y="693.8887">by the very same license.</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="707.8574"/><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="437" x="5" y="721.8262">This version of PlantUML records general local statistics about usage.</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="319" x="5" y="735.7949">(more info on http://plantuml.com/statistics-report)</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="749.7637"/><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="368" x="5" y="763.7324">Icons provided by OpenIconic :  https://useiconic.com/open</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="423" x="5" y="777.7012">Archimate sprites provided by Archi :  http://www.archimatetool.com</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="469" x="5" y="791.6699">Stdlib AWS provided by https://github.com/milo-minderbinder/AWS-PlantUML</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="467" x="5" y="805.6387">Stdlib Icons provided https://github.com/tupadr3/plantuml-icon-font-sprites</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="387" x="5" y="819.6074">ASCIIMathML (c) Peter Jipsen http://www.chapman.edu/~jipsen</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="416" x="5" y="833.5762">ASCIIMathML (c) David Lippman http://www.pierce.ctc.edu/dlippman</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="537" x="5" y="847.5449">CafeUndZopfli ported by Eugene Klyuchnikov https://github.com/eustas/CafeUndZopfli</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="387" x="5" y="861.5137">Brotli (c) by the Brotli Authors https://github.com/google/brotli</text><text fill="#000000" font-family="sans-serif" font-size="12" lengthAdjust="spacingAndGlyphs" textLength="0" x="9" y="875.4824"/><!--
```

Comment:
Why are we checking in GPLv3 stuff?!

Reply:
From http://plantuml.com/faq

> You can also use:
LGPL license
Apache license
Eclipse Public license
MIT license

Reply:
The .svg says this:
![image](https://user-images.githubusercontent.com/2642209/47882378-b2be1200-ddff-11e8-8438-6d9732a21819.png)

Reply:
I can upload the image as png instead of the svg then.

Reply:
Good catch @stephentoub. Lets remove the UML due to the licensing.

Reply:
Fixed the licensing to MIT instead.